events/filter.go

```diff
@@ -48,7 +49,8 @@ func Consume[T any](subscriber <-chan T, process func(event T)) jobs.Job {
 
 func recovery(errChan chan<- error) {
 	if r := recover(); r != nil {
-		errChan <- fmt.Errorf("job panicked:%s", r)
+		err := fmt.Errorf("job panicked: %s", errors.Wrap(r, 0).ErrorStack())
+		errChan <- err
```

Comment:
did you test this?
looks to me like you are suppressing the error message and only writing out the stack